docs/MILESTONES.md

```diff
@@ -0,0 +1,47 @@
+# Milestones Document
```

Comment:
@toteki We should instead move this to a google doc internally. Once agreed upon, we can create the Github milestone and corresponding issues 👍